scripts/convert_to_uvfits.py

```diff
@@ -17,8 +17,8 @@
 from astropy.time import Time
 
 # setup argparse
-a = argparse.ArgumentParser(description="A command-line script for converting a Miriad file to UVFITS format.")
-a.add_argument("files", type=str, nargs='*', help="Miriad files to convert to uvfits.")
+a = argparse.ArgumentParser(description="A command-line script for converting file(s) to UVFITS format.")
+a.add_argument("files", type=str, nargs='*', help="pyuvdata-compatible file(s) to convert to uvfits.")
```

Comment:
Consider adding an optional argument to specify the file type. This can help if the internal pyuvdata logic to decide this fails for some reason.

Reply:
Maybe also add an optional argument to pass the desired output filename.

Reply:
I'm hesitant to specify a filetype b/c the goal is to be filetype agnostic (using the `uvdata.read` function), _except_ for the replacement of the suffix which can be file-type specific, and would be benefitted if we specified an output filename as an option. So I think I'll add in the an optional output filename.

Reply:
The generic `uvdata.read` method has an optional argument for the filetype. It's useful in the case where the code has trouble figuring out what type of file the input file is. I was suggesting adding an optional parameter to this script that was piped to that optional argument. It's probably not needed often, so I'm ok with excluding it, but I wanted to clarify what I was talking about.

Reply:
oh got it, okay, no worries